<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MethodGui</class>
 <widget class="QWidget" name="MethodGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>35</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QComboBox" name="comboBoxDeclarationType"/>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxType"/>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditReturnType">
     <property name="placeholderText">
      <string>return type</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditName">
     <property name="placeholderText">
      <string>name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonParameters">
     <property name="text">
      <string>parameters</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
